airbyte-integrations/connectors/source-postgres-strict-encrypt/src/test/resources/expected_spec.json

```diff
@@ -54,11 +54,17 @@
         "airbyte_secret": true,
         "order": 5
       },
+      "jdbc_url_params": {
```

Comment:
I don't think this change in expected_spec.json is related to this PR. How did it end up here?

Reply:
> I don't think this change in expected_spec.json is related to this PR. How did it end up here?

Postgres Source Strict encrypt tests were broken in master branch, so i decided to fix them in scope of this PR